tests/resources/db/latest_schema.sql

```diff
@@ -17,11 +17,11 @@ CREATE TABLE experiments (
 
 
 CREATE TABLE registered_models (
-	name VARCHAR(256) NOT NULL,
-	creation_time BIGINT,
-	last_updated_time BIGINT,
-	description VARCHAR(5000),
-	CONSTRAINT registered_model_pk PRIMARY KEY (name),
+	name VARCHAR(256) NOT NULL, 
```

Comment:
extra spaces at the end of each line

Reply:
@zhidongqu-db Looks like there are still some trailing spaces here

Reply:
@dbczumar This is auto generated file and will be used in testing, the trailing spaces are expected

Reply:
@zhidongqu-db can we fix the autogeneration script to remove the trailing spaces?